sdk/core/core-util/src/bytesEncoding.browser.ts

```diff
@@ -72,6 +74,14 @@ export function uint8ArrayToUtf8String(uint8Array: Uint8Array): string {
   return dataString;
 }
 
+/**
+ * Decodes a Uint8Array into a hex string
+ * @internal
+ */
+export function uint8ArrayToHexString(uint8Array: Uint8Array): string {
+  return [...new Uint8Array(uint8Array)].map((x) => x.toString(16).padStart(2, "0")).join("");
```

Comment:
```suggestion
  return [...uint8Arra)].map((x) => x.toString(16).padStart(2, "0")).join("");
```

Reply:
or just `uint8Array.map`

Reply:
We need to convert to a normal array first since `map` on typed arrays must return another typed array